src/components/shared/links.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { Link } from "gatsby";
+import { GatsbyImage } from "gatsby-plugin-image";
+import { getHeadingLevel, isExternalURL } from "../../utils/ug-utils";
+import * as styles from "styles/links.module.css";
+
+function LinkElement(props) {
+  const url = props.url;
+
+  // Remove url from props to avoid passing it to the anchor or Gatsby Link
+  const copyProps = { ...props };
+  delete copyProps.url;
```

Comment:
I'm unsure as to the purpose of this, since it appears `copyProps` isn't used anywhere else. Can it be safely removed?

Reply:
That's my bad I actually meant to use copyProps and not props directly. The purpose of that is to create a shallow copy of the props and remove url from that copy so url doesn't get passed as an attribute because url is not a valid attribute for either a tags nor Gatsby Link.